docs/en/Deployment.md

```diff
@@ -0,0 +1,145 @@
+# Operation and maintenance management
+
+## Cluster building
+
+### 1 Single Master mode
+
+This is the simplest, but also the riskiest, mode that makes the entire service unavailable once the Broker restarts or goes down.Online environments are not recommended, but can be used for local testing and development.Here are the steps to build。
+
+**1）Start NameServer**
+
+```shell
+### Start Name Server first
+$ nohup sh mqnamesrv &
+ 
+### Then verify that the Name Server starts successfully
+$ tail -f ~/logs/rocketmqlogs/namesrv.log
+The Name Server boot success...
+```
+
+We can see 'The Name Server boot success.. ' in namesrv.log That indicates that the start NameServer has been successful.
```

Comment:
"that indicates the NameServer has been started successfully." will be more smoothly.